doc/start/quick-ceph-deploy.rst

```diff
@@ -40,13 +40,10 @@ Create a Cluster
 If at any point you run into trouble and you want to start over, execute
 the following to purge the configuration::
 
+	ceph-deploy purge {ceph-node} [{ceph-node}]
```

Comment:
> to purge the configuration

this is not accurate. @johnwilkins shall we reword it to something like, 

> to purge the Ceph package, and erase all its data and configurations::

?